lib/cli/src/generators/METEOR/index.js

```diff
@@ -43,6 +45,7 @@ export default async (npmOptions, { storyFormat = 'csf' }) => {
     `@storybook/react@${storybookVersion}`,
     `@storybook/addon-actions@${actionsVersion}`,
     `@storybook/addon-links@${linksVersion}`,
+    `@storybook/addon-knobs@${knobsVersion}`,
```

Comment:
Same as https://github.com/storybookjs/storybook/pull/9866/files#r379884879 for Mithril